content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md

```diff
@@ -86,6 +86,24 @@ from the table.
 - **Iterative Studio auto-hides irrelevant commits:** Iterative Studio
   identifies commits where metrics, files and hyperparameters did not change and
   hides them automatically.
+- **Iterative Studio auto-hides commits that contain `[skip studio]` in the
+  commit message:** This is particularly useful if your workflow creates
+  multiple commits per experiment and you would like to hide all those commits
+  except the final one. Below is an example:
+
+  > Suppose you submit hyper-parameter changes by creating a Git commit. Your CI
+  > job gets invoked when the commit is created, starting the model training
+  > process. Suppose your CI job creates a new Git commit with the experiment
+  > results at the end of model training. This new Git commit, therefore,
+  > contains the new values of the hyper-parameters as well as experiment
+  > results (metrics).
+  >
+  > In Iterative Studio, you may want to display only this final commit and
+  > auto-hide the original commit which contains the the new values of the
+  > hyper-parameters but not the experiment results (metrics). In this case,
+  > what you should do is add the string `[skip studio]` to the commit message
+  > when you create the original commit.
```

Comment:
This is really verbose IMO, and the block quote is not very clarifying anyway. I would also not call this "automatic" if it's done via a commit message. Are the examples realistic (based on real-life user cases?) Consider something like:

```suggestion

- **Iterative Studio** hides commits that contain `[skip studio]` in their
  message. This is useful when your workflow requires extra commits which
  you do not want to cause another CI job run.
```

Reply:
This isn't about CI job run. It's about hiding the extra commits in Studio. Also, it's ok to reduce the amount of text, but we do need an example.

Reply:
Will people really read that example in a huge quote block? Maybe there's a better place to have the example and then just link from here.